.github/workflows/functional_test_v2.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
           - configuration_switching
           - istio
     runs-on: ubuntu-latest
+    continue-on-error: ${{ !contains(github.event.pull_request.labels.*.name, 'Ignore Tests') }}
```

Comment:
Shouldn't the condition be flipped here? We want to continue on error if the PR contains the label.

Reply:
Yeah, looks like it should be flipped.

Can non org members add labels on PRs btw?

Reply:
Github workflow parameters like this use passed values to functions in a syntax ordering that seems backwards.
Other Examples:
- https://github.com/signalfx/splunk-otel-collector-chart/blob/2486516e3785ff4a9821c9307e07edc2de951668/.github/workflows/validate-changelog.yaml#L12
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/workflows/changelog.yml#L49
- Non-org members cannot at labels.

Reply:
Going to add some Github label descriptions for this new label and previous labels that are missing descriptions.

Reply:
The `continue-on-error` config determines whether the job should continue running even if a step fails. My understanding is that your intention with this PR is to let the action continue on error if the `Ignore Tests` label is set. The condition should be reversed by removing the `!` operator in that case (we want the config value to be true when the label exists).

In the quoted validate-changelog.yaml file `if` condition, it ensures that the job runs only when the `Skip Changelog` label is not present and the pull request title does not contain `[chore]`. I am not sure I understand what you mean by backwards ordering.

Reply:
> Github workflow function like this use parameter ordering that seems backwards, but this is correct.

Just to clarify, is this in reference to the condition comment? The examples you shared are correct because the logic is "If the label is not applied, validate the changelog exists and is valid." In this case, we're saying "If the label is applied, we want to continue on error". So the goal of the label is to have a check of "If true, return true", at least the way it's worded.

Feel free to clarify further, I may be misunderstanding something here.

Reply:
Okay now I see what you folks are talking about. My apologies, between my github workflow and bash code related work lately I can interpret the initial comments in several ways.
- I can make updates for this because you folks are right. I might ask for a explicit/specific code suggestions if possible in the future, granted, this one was mostly on me.
- I need to test changes with several other non-trivial separate code branches involved at the same time so I'm lagging on testing sometimes.